ha/setup/ha_setup.py

```diff
@@ -340,7 +341,6 @@ def process(self):
         self._update_cluster_manager_config()
 
         # Update cluster and resources
-        self._cluster_manager = CortxClusterManager(default_log_enable=False)
```

Comment:
This is required in this position only

in `__init__` some config are not yet done like
self._update_cluster_manager_config()

Reply:
Right. I got that. Changing it now.